flight/PiOS/Common/pios_mpu6000.c

```diff
@@ -51,6 +56,17 @@
 #endif
 #define MPU6000_SPI_LOW_SPEED               1000000
 
+// Private variables
```

Comment:
I thought with the improvement of SPI Speed it could maybe work on CC3D when at first dropped down-sampling from 8kHz to a lower rate, and then filtering down-sampling to 1kHz or below might work.
But as you say it will consume some RAM, so its better to disable that.
I also got now more information about the DLPF implementation in the MPU, so when using max 1kHz output rate for control loop, the new CIC decimation might not have a big advantage.